kokoro/continuous.bat

```diff
@@ -2,7 +2,7 @@
 
 cd github/jib
 
-cd jib-core && call gradlew.bat clean build --info
+cd jib-core && call gradlew.bat clean build publishToMavenLocal --info
```

Comment:
I think I remember the problem with using the "publish" plugin vs "maven" plugin. It's that the publish plugin didn't behave when trying to publish to maven central, but it's now irrelevant since we'll be using kokoro to build and deploy.

Reply:
Yeah, this is only for jib-core. Is there another way for jib-maven-plugin to use jib-core as a dependency but not have it be used as a Maven artifact dependency?